docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
-# sha256 as of 2020-09-25 for mainline-alpine
-FROM nginx@sha256:4635b632d2aaf8c37c8a1cf76a1f96d11b899f74caa2c6946ea56d0a5af02c0c
+# sha256 as of 2021-08-18 for nginx:mainline-alpine
```

Comment:
I've been moving toward putting the tag in the `FROM` line, a la `nginx:mainline-alpine@sha256:...` and then having only the datestamp in the comment. (Didn't know you could use that syntax until recently!)

Reply:
TIL! That's much clearer, will do.